ibc/chain.go

```diff
@@ -69,11 +69,8 @@ type Chain interface {
 	// get the fees in native denom for an amount of spent gas
 	GetGasFeesInNativeDenom(gasPaid int64) int64
 
-	// GetPacketAcknowledgement fetches ibc packet ack or an error if not found
-	GetPacketAcknowledgement(ctx context.Context, portID, channelID string, seq uint64) (PacketAcknowledgement, error)
-
-	// GetPacketSequence returns the packet sequence given the transaction's hash
-	GetPacketSequence(ctx context.Context, txHash string) (uint64, error)
+	// Acknowledgements returns all acknowledgements in a block at height
+	Acknowledgements(ctx context.Context, height uint64) ([]PacketAcknowledgement, error)
```

Comment:
Originally I had it return just 1 ack in this method vs returning a slice. 

However, given blocks may have many transactions, this may not fit all cases, especially given parallel tests. Totally possible multiple acks are in a block.